### PR TITLE
add partners section and react paris

### PR DESCRIPTION
**Pull request by JeanneGrenet**
## Description : 
Add a partner section in the homepage 

## Screen with 2 partners : 
I only put React Paris one time but it's for you to see with more partners
image

### CODE REVIEW

**Review by charlelisefouasse — APPROVED** [review]

File: public/partners/react-paris.png
Comment (yoannfleurydev) [inline public/partners/react-paris.png:1]: I think you didn't used the same template as the sponsors. We should to keep it homogeneous

File: src/app/(homepage)/page.tsx [hunk 1]
```diff
@@ -15,6 +16,7 @@ export default function Home() {
       <Events />
       <Features />
       <Organizers />
+      <Partners />
       {/* <Sponsors /> */}
```
Comment (yoannfleurydev) [inline src/app/(homepage)/page.tsx:20]: We will remove the `<Sponsors />` comment by the way

File: src/app/(homepage)/partners.tsx [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import Image from "next/image";
+
+export async function Partners() {
+  const allPartners = await collections.partner.getAll();
+  return (
+    <div className="bg-white py-20">
+      <div className="mx-auto max-w-7xl px-6 lg:px-8">
+        <div className="mx-auto max-w-2xl lg:max-w-none">
+          <h2 className="font-heading text-3xl font-bold uppercase text-gray-900 sm:text-4xl">
+            Meet our partners
```
Comment (yoannfleurydev) [inline src/app/(homepage)/partners.tsx:12]:
```suggestion
            Partners
```

File: src/app/(homepage)/partners.tsx [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+          </h2>
+          <ul className="mx-auto mt-10 grid max-w-2xl grid-cols-2 gap-x-6 gap-y-8 sm:grid-cols-4 lg:mx-0 lg:max-w-none lg:grid-cols-4">
+            {allPartners.map((partner) => (
+              <li key={partner.name} className="flex flex-col gap-4">
```
Comment (yoannfleurydev) [inline src/app/(homepage)/partners.tsx:16]: I'm not sure about the CSS here. There is only 1 element (the image).

File: src/app/(homepage)/partners.tsx [hunk 4]
```diff
@@ -0,0 +1,37 @@
...
+                  href={partner.href ?? "#"}
+                  title={partner.name}
+                  target="_blank"
+                  rel="noreferer"
+                >
+                  <Image
+                    src={partner.image.src}
+                    alt={partner.image.alt}
+                    width={500}
+                    height={500}
```
Comment (yoannfleurydev) [inline src/app/(homepage)/partners.tsx:27]: After using the template on the image, you will have the good values for those.

File: src/app/(homepage)/partners.tsx [hunk 5]
```diff
@@ -0,0 +1,37 @@
...
+      <div className="mx-auto max-w-7xl px-6 lg:px-8">
+        <div className="mx-auto max-w-2xl lg:max-w-none">
+          <h2 className="font-heading text-3xl font-bold uppercase text-gray-900 sm:text-4xl">
+            Meet our partners
+          </h2>
+          <ul className="mx-auto mt-10 grid max-w-2xl grid-cols-2 gap-x-6 gap-y-8 sm:grid-cols-4 lg:mx-0 lg:max-w-none lg:grid-cols-4">
+            {allPartners.map((partner) => (
+              <li key={partner.name} className="flex flex-col gap-4">
+                <Link
+                  href={partner.href ?? "#"}
```
Comment (yoannfleurydev) [inline src/app/(homepage)/partners.tsx:18]: In the schema, URL is optional. We don't put the link if there is not href. `#` should not be in production.

**Review by yoannfleurydev — CHANGES_REQUESTED** [review]

File: src/app/(homepage)/partners.tsx (hunk 3 above)
Comment (JeanneGrenet) [reply src/app/(homepage)/partners.tsx:16]: There will be other images with other partners no ?

Comment (yoannfleurydev) [reply src/app/(homepage)/partners.tsx:16]: Yeah, but each partner is in an `li`. `flex flex-col` won't be useful for just an image.

**Review by ivan-dalmet — APPROVED** [review]

**Pull request merged**